.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Continuous Integration
+
+on: [ push ]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+
+    strategy:
+      matrix:
+        go: [ '1.13' ]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go }}
+
+      - run: gofmt -l $(find . -type f -name '*.go' -not -path "./vendor/*") 2>&1
```

Comment:
Do we have access to `goimports` in GitHub Actions? If so, it'd be better to use that. A while ago we made the decision to use that for formatting, since it separately groups the official and third-party import packages, and removes unused imports (otherwise it's the same as `gofmt`).

@camh- and/or @juliaogris might also want to use `golangci-lint`, if it's available?

Also, there's no `vendor` directory in the Alpaca codebase (we're using Go Modules) so you don't need the `-not -path "./vendor/*"`.